samples/excel/30-events/events-workbook-and-worksheet-collection.yaml

```diff
@@ -0,0 +1,257 @@
+order: 7
+id: excel-events-workbook-and-worksheet-collection
+name: Workbook and WorksheetCollection events
+description: 'Registers event handlers for worksheet addition, activation, deactivation, and settings changes.'
```

Comment:
For consistency with other snippet descriptions, change to -> `Registers event handlers that run when a worksheet is added, activated, or deactivated, or worksheet settings are changed.`